client/src/Components/Videos.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import videosData from "./exampleresponse.json";
+import VideoCard from "./VideoCard";
+import NewVideo from "./NewVideo";
+
+function Videos({ newVideos, handleSubmit }) {
+  const [videos, setVideos] = useState(videosData);
+
+  const handleDelete = (id) => {
+    setVideos((prevVideosData) =>
+      prevVideosData.filter((video) => video.id !== id)
+    );
+  };
+
+  const addVideos = (video) => {
+    console.log("this is video", video);
+    setVideos([video, ...videos]);
+    console.log("in add videos", videos);
+  };
```

Comment:
Good job in useing `spread operator`! can you also remove unused codes in your PR? `console.log`s